src/Relude/DeepSeq.hs

```diff
@@ -26,27 +26,58 @@ import Relude.Monad (MonadIO, liftIO, (<$!>))
 
 import qualified Control.Exception.Base (evaluate)
 
+{- | Lifted alias for 'Control.Exception.Base.evaluate' with clearer name.
 
--- | Lifted alias for 'Control.Exception.Base.evaluate' with clearer name.
+>>> :sprint list
+list = _
+>>> () <$ evaluateWHNF list
+>>> :sprint list
+list = _ : _
+
+-}
 evaluateWHNF :: MonadIO m => a -> m a
 evaluateWHNF = liftIO . Control.Exception.Base.evaluate
 {-# INLINE evaluateWHNF #-}
 {-# SPECIALIZE evaluateWHNF :: a -> IO a #-}
 
--- | Like 'evaluateWNHF' but discards value.
+{- | Like 'evaluateWNHF' but discards value.
+
+>>> :sprint list
+list = _
+>>> evaluateWHNF_ list
+>>> :sprint list
+list = _ : _
+
+-}
 evaluateWHNF_ :: MonadIO m => a -> m ()
 evaluateWHNF_ what = (`seq` ()) <$!> evaluateWHNF what
 {-# INLINE evaluateWHNF_ #-}
 {-# SPECIALIZE evaluateWHNF_ :: a -> IO () #-}
 
--- | Alias for @evaluateWHNF . force@ with clearer name.
+{- | Alias for @evaluateWHNF . force@ with clearer name.
+
+>>> :sprint list
+list = _
+>>> () <$ evaluateNF list
+>>> :sprint list
+list = ??? unsure which list we're using in example
```

Comment:
I fixed my comment last time to specify the list for the example. The same list can be used in each example